src/cpu/flexcpu/inflight_inst.cc

```diff
@@ -52,11 +53,62 @@ InflightInst::InflightInst(ThreadContext* backing_context,
     _pcState(pc_)
 {
     _timingRecord.creationTick = curTick();
+    // Tick's are unsigned, so effectively, -1 is MAX_INT
+    _timingRecord.decodeTick = -1;
```

Comment:
Can you put this in the header as the initial values for the `_timingRecord` struct?

Also, IIRC there's a macro for `MAX_TICK` that would be better to use here.

Reply:
I'll look into MAX_TICK.

Reply:
The macro is `MaxTick = ULL(0xffffffffffffffff)`, where `ULL(N)` is `((uint64_t)N##ULL)`.

Reply:
I suggest leaving it as `MaxTick` in the trace file but rendering it as "--" or something like that in the trace viewer.